controllers/drill/job.go

```diff
@@ -41,8 +41,8 @@ func NewJob(cr *perfv1alpha1.Drill, configMap *corev1.ConfigMap) *batchv1.Job {
 		Namespace: cr.Namespace,
 	}
 
-	cmdLineArgs := fmt.Sprintf("%s --benchmark %s", cr.Spec.Options, cr.Spec.BenchmarkFile)
-	command := fmt.Sprintf("cd %s && %s %s", benchmarksDir, drill, cmdLineArgs)
+	Args := fmt.Sprintf("%s --benchmark %s", cr.Spec.Args, cr.Spec.BenchmarkFile)
```

Comment:
this is confusing: we take cr.Spec.Args plus BenchmarkFile and call it 'Args'.
I'd prefer not to reuse the same variable with different content. Also, 'Args' should be lowercased.